### PR TITLE
feat(iot-hub-device-update): added command to remove updates from iot-hub

### PR DESCRIPTION
**Pull request by JanZachmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,16 @@ omnect-cli iot-hub-device-update import-update --help
...
 
+### Remove update from IoT Hub
+This command removes an update from Azure Device Update for IoT Hub that was previously imported by `import-update` command.
+
+Detailed description:
+```sh
+omnect-cli iot-hub-device-update remove-update --help
+```
+
+**Note**: The import process may take several minutes.
```
Comment (JoergZeidler) [inline README.md:113]: .... The remove process may take serveral minutes

File: README.md [hunk 2]
```diff
@@ -102,6 +102,16 @@ omnect-cli iot-hub-device-update import-update --help
 
 **Note**: The import process may take several minutes.
 
+### Remove update from IoT Hub
+This command removes an update from Azure Device Update for IoT Hub that was previously imported by `import-update` command.
```
Comment (JoergZeidler) [inline README.md:106]: just a notice: omnect-cli could remove updates that are created manually...
->  delete? "that was previously imported by `import-update` command"

File: Cargo.toml [hunk 3]
```diff
@@ -48,6 +48,7 @@ tokio = { version = "1", features = [
     "io-util",
     "fs",
     "net",
+    "rt-multi-thread"
```
Comment (empwilli) [inline Cargo.toml:51]: No hard blocker for me, just out of curiosity: why do we need the parallel runtime for the CLI?

Comment (JanZachmann) [reply Cargo.toml:51]: My guess is that they changed the feature defaults in tokio 1.37.0. At least I saw this error after running `cargo update`:
```
error: The default runtime flavor is `multi_thread`, but the `rt-multi-thread` feature is disabled.
   --> src/lib.rs:255:13
    |
255 |             #[tokio::main]
    |             ^^^^^^^^^^^^^^
    |
    = note: this error originates in the attribute macro `tokio::main` (in Nightly builds, run with -Z macro-backtrace for more info)
```

File: README.md (hunk 1 above)
Comment (JanZachmann) [reply README.md:113]: My tests didn't show any delays.

File: README.md (hunk 2 above)
Comment (JanZachmann) [reply README.md:106]: I just want to document workflows in context of omnect-cli. In this case the reader should just know that he has to import an update before he can remove one.

File: src/device_update.rs [hunk 4]
```diff
@@ -265,12 +264,44 @@ pub async fn import_update(
...
+}
+
+#[allow(clippy::too_many_arguments)]
+#[tokio::main]
+pub async fn remove_update(
+    tenant_id: String,
+    client_id: String,
+    client_secret: String,
+    instance_id: String,
+    device_update_endpoint_url: &Url,
```
Comment (empwilli) [inline src/device_update.rs:285]: Nit: i guess we could pass per reference here?

**Review by empwilli — APPROVED** [review]
Looks fine to me

File: README.md (hunk 1 above)
Comment (JanZachmann) [reply README.md]: done

**Review by JoergZeidler — APPROVED** [review]

**Review by empwilli — APPROVED** [review]

**Pull request merged**